wifiphisher/common/phishingpage.py

```diff
@@ -336,8 +369,15 @@ def find_user_templates(self):
             # check to see if it is a directory and not in the database
             if (os.path.isdir(os.path.join(self._template_directory, name)) and
                     name not in self._templates):
-                # add it to the list
-                local_templates.append(name)
+                # check for template
+                result = self.search_template(name)
+                # if we got template directory name, then...
+                if result == name:
+                    # just add it to the list
```

Comment:
it's not so straightforward to use `result == name` to judge this is a valid template.

Reply:
Accompany with my previous comment:

```python
is_valid, output = self.is_valid_template(name)
if not is_valid:
    print "[" + constants.R + "!" + constants.W + "] " + output + name 
```